sdk/table/azure-table/README.md

```diff
@@ -0,0 +1,354 @@
+# Azure Table client library for Python
+
+Azure Table is a service for storing data that can be accessed from anywhere in the world via authenticated calls using HTTP or HTTPS. 
+
+Common uses of Azure Table include:
+
+* Storing structured data in the form of tables
+* Querying tables and entities 
+
+[Source code](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-queue/azure/storage/queue) | [Package (PyPI)](https://pypi.org/project/azure-storage-queue/) | [API reference documentation](https://aka.ms/azsdk-python-storage-queue-ref) | [Product documentation](https://docs.microsoft.com/azure/storage/) | [Samples](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-queue/samples)
+
+## Getting started
+
+### Prerequisites
+* Python 2.7, or 3.5 or later is required to use this package.
+* You must have an [Azure subscription](https://azure.microsoft.com/free/) and an
+[Azure storage account](https://docs.microsoft.com/azure/storage/common/storage-account-overview) to use this package.
+
+### Install the package
+Install the Azure Table client library for Python with [pip](https://pypi.org/project/pip/):
+
+```bash
+pip install azure-table
+```
+
+### Create a storage account
+If you wish to create a new storage account, you can use the
+[Azure Portal](https://docs.microsoft.com/azure/storage/common/storage-quickstart-create-account?tabs=azure-portal),
+[Azure PowerShell](https://docs.microsoft.com/azure/storage/common/storage-quickstart-create-account?tabs=azure-powershell),
+or [Azure CLI](https://docs.microsoft.com/azure/storage/common/storage-quickstart-create-account?tabs=azure-cli):
+
+```bash
+# Create a new resource group to hold the storage account -
+# if using an existing resource group, skip this step
+az group create --name my-resource-group --location westus2
+
+# Create the storage account
+az storage account create -n my-storage-account-name -g my-resource-group
+```
+
+### Create the client
+The Azure Table client library for Python allows you to interact with three types of resources: the storage
+account itself, tables, and entities. Interaction with these resources starts with an instance of a [client](#clients).
```

Comment:
should this be two types of resources? the account and tables? since I don't believe we have an EntityClient.